packages/cli/src/commands/build.ts

```diff
@@ -678,28 +678,16 @@ export async function runPackageJsonScript(
 
 async function linkOrCopy(existingPath: string, newPath: string) {
   try {
-    if (
-      newPath.endsWith('.nft.json') ||
```

Comment:
@AndyBitz we actually want to keep this copying here logic. it fixes #6975

Reply:
Ah, good point